src/Quartz.Extensions.DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -20,6 +20,17 @@ public static class ServiceCollectionExtensions
         public static IServiceCollection AddQuartz(
```

Comment:
Now that Quartz.NET depends on `Microsoft.Extensions.Options`, I think it would be better to call `services.AddOptions()` to ensure the options services are automatically imported.

Reply:
Added.